source/query-bar.txt

```diff
@@ -57,7 +57,7 @@ operator.
    Builder to enter a query into the query bar. For details, see
    :ref:`build-query`.
 
-As you type, the :guilabel:`Apply` button is disabled and the
+As you type, the :guilabel:`Find` button is disabled and the
```

Comment:
We usually don't specify the UI element unless it is unclear which element you mean (i.e. two elements named "Find": one link and one button).